### DISCUSSION TITLE
Multiple LDAP authentication methods conflict each other

### DISCUSSION THREAD

**Discussion by garykuok** [requarks/wiki#4660] [Error / Bug Report]
Hello,

I'm setting up the LDAP authentication for Wiki.js. I notice that if I create multiple LDAP methods, none of them will work. 
If I only create one among them, then it will work as expected.

I think there could be some database / login mapping caused this behavior? I'm using docker-compose to run Wiki.js and postgre-sql in two images in one yaml.

Best,
Gary

**Comment by Pavel68-ru** [comment]
Hello.
I'm setting up LDAP authentication in a js wiki with two domains. When creating the second LDAP, the first one stops working, and the last one that has been changed works.
Installed on Ubuntu postgresql database.
Is there a solution to this problem?

**Reply by garykuok** [reply]
Hello, we also encountered exactly the same behaviour. We don't have a solution currently, but just stick to one LDAP method...

**Reply by Pavel68-ru** [reply]
Hello. It follows from the behavior that 2 records conflict and change each other's settings. In which the last one made changes, it works.

**Reply by garykuok** [reply]
Yes, but can you make the two LDAP methods work in parallel? Our web couldn't.
